src/WorkBC.Tests/appsettings.json

```diff
@@ -1,6 +1,7 @@
 {
   "ConnectionStrings": {
-    "ElasticSearchServer": "http://localhost:9200"
+    "ElasticSearchServer": "http://localhost:9200",
+    "DefaultConnection": "Server=localhost;integrated security=true;Database=WorkBC_JobBoard_DEV"
```

Comment:
Ideally, we'd be using an `:memory:` database instead of the MSSQL database, but there are subtle differences between SQLite and MSSQL that we may want to catch in testing.